.github/PULL_REQUEST_PROCESS.md

```diff
@@ -0,0 +1,41 @@
+# DNN Platform Pull Request Proceseses
+
+The goal of this document is to set standards for the review, processing, and approval of all Pull requests into the DNN_Platform GitHub repository, as well as associated commmunity open-source repositories.  These rules apply to ALL pull requests, regardless of the submitter, or complexity of the change.
+
+## Prerequisites
+The following items must all be true prior to a pull request being submitted to the DNN_Platform, if any of the following items are not true the pull request will be returned for edits referencing the specific missing item(s).
+
+* Change should represent an entire solution for the issue at hand.  Partial requests will NOT be processed.
+* Change should have a supporting issue logged on the DNN_Platform GitHub account, documenting the issue resolved, following the procedures outlined on the [Contribute Page](CONTRIBUTING.md)
+* Exception: Security items can be addressed individually, and should be initially communicated to the security@dnnsoftware.com email address for coordination
+* Pull request comment should contain at a minimum the following details (When creating, a default template will also prompt you for the proper information)
+--* Issue #, referenced such as “Fixes #24”
+--* Release Note: Suggested release note for the change such as “Improvement of user profile to improve performance when searching.”
+--* Testing Steps: Suggested testing steps for validation of the change
```

Comment:
Let's take the occasion to add *Testing steps* to issue template and *Fixes #XXX* to the PR template